src/components/PlioQuestion.vue

```diff
@@ -119,6 +131,12 @@ export default {
     // Returns the index of the question that has popped up.
     currentQuestionIndex() {
       return this.plioQuestion.id;
+    },
+    reviseButton() {
+      return document.getElementById('revise-button')
+    },
+    skipButton() {
+      return document.getElementById('skip-button')
```

Comment:
Added these as computed properties as the call was being repeated in the code and the browser had to go and find it again and again